### DISCUSSION TITLE
How to auto delete similar photos?

### DISCUSSION THREAD

**Discussion by DenuxPlays** [cloud-py-api/mediadc#158] [Q&A]
Hey,
How can I auto or mass delete all duplicates?
I currently have over 100 duplicate groups and I do not want to look through every single one.
Is there a way to auto delete all duplicates or something like this?

**Comment by karel-haruda-abra** [comment]
I have a similar problem, I would like simple rules for deleting duplicates. For example, delete those with worse resolution, prioritize deleting / keeping photos in a certain path, or a simple python script that receives information about duplication, 2..n records where each record describes one duplicate file (resolution, file size, dimensions in px, location) and decides which ones you want to keep and which ones to delete.
